docs/guides/route_filtering/inbound/bogon_prefixes.md

```diff
@@ -119,30 +119,27 @@ In IPv6, there is a [similar list at IANA](http://www.iana.org/assignments/ipv6-
       127.0.0.0/8+,       # RFC 1122 Loopback
       169.254.0.0/16+,    # RFC 3927 Link Local
       172.16.0.0/12+,     # RFC 1918 Private
-      192.0.0.0/24+,      # RFC 6890 Protocol Assignments
       192.0.2.0/24+,      # RFC 5737 Documentation TEST-NET-1
-      192.88.99.0/24+,    # RFC 7526 6to4 anycast relay
       192.168.0.0/16+,    # RFC 1918 Private
       198.18.0.0/15+,     # RFC 2544 Benchmarking
       198.51.100.0/24+,   # RFC 5737 Documentation TEST-NET-2
       203.0.113.0/24+,    # RFC 5737 Documentation TEST-NET-3
       224.0.0.0/4+,       # RFC 5771 Multicast
       240.0.0.0/4+        # RFC 1112 Reserved
     ];
-    define BOGON_PREFIXES6 = [ ::/8+ ];
-        ::/128,          # RFC4291 Unspecified Address
-        ::1/128,         # RFC4291 Loopback Address
-        ::ffff:0:0/96+,  # RFC4291 IPv4-mapped Address
-        64:ff9b::/96+,   # RFC6052 IPv4-IPv6 Translat.
-        64:ff9b:1::/48+, # RFC8215 IPv4-IPv6 Translat.
```

Comment:
Why are you removing these?

Reply:
I reduced the prefixes to common ground

64:ff9b:: and 6to4 was dropped in other sources
I also see possible valid routes for TEREDO and 6to4 in the DFZ.